packages/docs/docs/sidekicks-quickstart.md

```diff
@@ -27,212 +34,193 @@ At the end of this quickstart you will:
 
 :::warning Prerequisites
 Before you get started, you will need to have a free Fixie developer account and have some tools installed on your machine. You will also need a text editor. If you don't have a preferred text editor, we recommend [Visual Studio Code](https://code.visualstudio.com/).
-
-If you don't want to fuss with installing things on your machine, you can sign-up for a free DevZero account and use the pre-configured template for Fixie Sidekicks.
 :::
 
 :::tip On Windows? Use WSL.
 If you are using a Windows machine, we highly recommend using the Windows Subsystem for Linux (WSL) for development with Node.js. This is optional. If you want to use WSL, follow [this guide](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl) which will get you set-up with WSL, Node, and VSCode (for your text editor).
 :::
 
-### a) Sign-up for Fixie
+### a) Get a Fixie developer account
 
-We will use [Fixie](https://fixie.ai) for hosting our Sidekick. Sign-up for a free Fixie developer account:
+We will use [Fixie](https://fixie.ai) for hosting and managing our Sidekick.
+Sign up for a free Fixie developer account:
 
-1. Go to the [Fixie sign-up page](https://console.fixie.ai).
+1. Go to the [Fixie console page](https://console.fixie.ai).
 1. Create an account using either a Google or GitHub account.
 1. Navigate to your [profile page](https://console.fixie.ai/profile).
 
 <img src={FixieProfileAPIKey} alt="Fixie profile page where you can get your API key." width="600"/>
 
-Keep this page open as we will come back to this in Step 2 to grab our API key which is required for deployment.
+The Fixie Console is where you will test and manage your Sidekick once it's built.
+For now, the main thing you need is your Fixie API Key, which is found on your
+[profile page](https://console.fixie.ai/profile).
 
 ### b) Install Node.js
 
-Install the current [LTS version of Node.js](https://nodejs.org/en).
+Sidekicks are based on AI.JSX, which in turn relies on Node.js, a JavaScript
+runtime. Install the current [LTS version of Node.js](https://nodejs.org/en).
 
 ### c) Install the Fixie CLI
 
-In your terminal execute the following:
+The Fixie command-line interface is provided by the [fixie](https://www.npmjs.com/package/fixie) package in NPM. Install it by running:
 
 ```terminal
-npm install -g @fixieai/fixie@latest
+$ npm install -g fixie@latest
```

Comment:
I think this is actually not what we want. The code block has a built-in copy button. If we add a `$` to the beginning of the line, then the user can't actually copy/paste directly into their terminal. They have to take the `$` back out.

<img width="997" alt="image" src="https://github.com/fixie-ai/ai-jsx/assets/829827/58e4489a-12f9-4b05-a27f-0ee681a91fd8">

Reply:
incorporating this change. good point!